src/extension.ts

```diff
@@ -69,23 +69,36 @@ function didOpenTextDocument(
   if (!folder) {
     return;
   }
+
   if (
+    workspace
+      .getConfiguration()
```

Comment:
`workspace.getConfiguration` is also called later on, maybe consider storing it into a `const configuration` and reusing the same variable ?

Reply:
ye, the problem is the function was clearly never meant to take configuration arguments. However it would take some re-architecture to make it nice, and I didn't want to push for too many changes.